config/docker/cros-baseline/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM debian:bullseye-slim
+MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    apt-transport-https \
+    ca-certificates \
+    ssh
```

Comment:
I think `ssh` should ideally be in the list below, with `git` and `ping`. This part about certificates is more about being able to use HTTPS apt sources.  It's not a big issue, but keeping it the same as other Docker images can help with caching.

Reply:
Moved